nav2_msgs/action/FollowGPSWaypoints.action

```diff
@@ -0,0 +1,9 @@
+#goal definition
+# geographic_msgs/GeoPose[] gps_poses
+geographic_msgs/GeoPose[] gps_poses
```

Comment:
@SteveMacenski, we changed this to be a [geographic_msgs/GeoPose](http://docs.ros.org/en/api/geographic_msgs/html/msg/GeoPose.html) instead of the [custom message](https://github.com/ros-planning/navigation2/pull/2111/files#diff-c84bbc75a03d44dabbe17b0347390ac22c32e912e06e827a12d654c6e05f6fa5)  that was created in the previous PR. Is that ok or should we keep it as it was before?

Reply:
LGTM, I didn't know that existed already

Reply:
Done